<?xml version="1.0" encoding="utf-8"?>
<report
	progress="">
	<activity
		result="Failed"
		duration="13.72s"
		durationms="13718"
		type="root"
		rid="3411ad374709bd0"
		totalerrorcount="1"
		totalwarningcount="0"
		totalsuccesstestcasecount="0"
		totalfailedtestcasecount="0"
		totalblockedtestcasecount="0"
		childsuccesscount="0"
		childfailedcount="0"
		childblockedcount="0"
		totalsuccesscount="0"
		totalfailedcount="0"
		totalblockedcount="0"
		totalmaintenancemodecount="0"
		user="Fahad Shah"
		rxversion="8.0.0.0"
		host="HUAWEI LUA-U22 (Unknown)
WLAN - 192.168.3.11"
		osversion="-"
		runtimeversion="-"
		procarch="-"
		oslanguage="-"
		dtnlanguage="-"
		screenresolution="-"
		timestamp="2/23/2018 12:09:33 PM"
		timestampiso="2018-02-23T07:09:33"
		endtime="2/23/2018 12:09:47 PM"
		timeoutfactor="1"
		module="VisitOtherShopOptions">
		<detail>
			VisitOtherShopOptions
		</detail>
		<varbindings>
		</varbindings>
		<item
			timeRelativeToTestSuiteStartTime="00:13.718"
			timeRelativeToTestModuleStartTime="00:13.718"
			timeWallClock="12:09:47"
			time="00:13.718"
			level="Error"
			category="Module">
			<message>
				The test run has been aborted by the user.
			</message>
			<metainfo
				loglvl="Error">
			</metainfo>
		</item>
	</activity>
</report>
